src/handlers/dns/v2/DnsStreamHandler.cpp

```diff
@@ -514,76 +487,124 @@ void DnsMetricsBucket::specialized_merge(const AbstractMetricsBucket &o, Metric:
     const auto &other = static_cast<const DnsMetricsBucket &>(o);
 
     // rates maintain their own thread safety
```

Comment:
remove comment

Reply:
it was missing a metric. I've added the metric there